test/serverTest.py

```
@@ -0,0 +1,3 @@
+import requests
+import json
+print(requests.get("http://localhost:8080/health").text)
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)